portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="contact.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="contact_content">
```

Comment:
Change to contact-content (with a hyphen instead of an underscore).

Reply:
Changed to contact-content